vaadin-crud-flow-parent/vaadin-crud-flow/pom.xml

```diff
@@ -82,7 +82,7 @@
         <dependency>
             <groupId>org.springframework.data</groupId>
             <artifactId>spring-data-commons</artifactId>
-            <version>3.4.1</version>
+            <version>${spring.boot.version}</version>
```

Comment:
This is incorrect, it just happens to match in this version. If you want to use spring boot version, then you need to import the BOM in the parent pom and omit the version declaration here

Reply:
The easier solution would be to rename `spring.boot.version` to `spring-data-commons.version`

Reply:
> The easier solution would be to rename spring.boot.version to spring-data-commons.version

Then @ZheSun88 needs to update that version individually when bumping the version for the platform. Would that be OK?

Reply:
Changed to `spring-data-commons.version`  for now.